batchflow/models/torch/base.py

```diff
@@ -1751,11 +1755,26 @@ def save(self, path, use_onnx=False, path_onnx=None, use_openvino=False, path_op
 
             ov.save_model(model, output_model=path_openvino)
 
-            # Save the rest of parameters
-            preserved = self.PRESERVE_OPENVINO - ignore_attributes
+            if pickle_metadata:
+                # Save the rest of parameters
+                preserved = self.PRESERVE_OPENVINO - ignore_attributes
+                preserved_dict = {item: getattr(self, item) for item in preserved}
+                torch.save({'openvino': True, 'path_openvino': path_openvino, **preserved_dict},
+                        path, pickle_module=pickle_module, **kwargs)
+
+        elif use_safetensors:
+            from safetensors.torch import save_file
+            state_dict = self.model.state_dict()
+
+            path_safetensors = path_safetensors or (path + '.safetensors')
+            save_file(state_dict, path_safetensors)
```

Comment:
In the case when I want to save only safetensors model I can
1) `model.save(path='model.pt', use_safetensors=True, path_safetensors='model.safetensors', pickle_metadata=False)` -> in that case I don't use `path` which is necessary argument and i can't skip it
2) `model.save(path='model.pt', use_safetensors=True, pickle_metadata=False)` -> the model will be in `model.pt.safetensors`, I am not sure that two extensions is a good idea

Think how to avoid it. For example, make `path=None`, by default.